test/flycheck-test.el

```diff
@@ -3044,14 +3044,26 @@ See https://github.com/flycheck/flycheck/issues/531 and Emacs bug #19206"))
 
 (flycheck-ert-def-checker-test erlang erlang error
   (flycheck-ert-should-syntax-check
-   "language/erlang/error.erl" 'erlang-mode
+   "language/erlang/erlang/error.erl" 'erlang-mode
    '(7 nil error "head mismatch" :checker erlang)))
 
 (flycheck-ert-def-checker-test erlang erlang warning
   (flycheck-ert-should-syntax-check
-   "language/erlang/warning.erl" 'erlang-mode
+   "language/erlang/erlang/warning.erl" 'erlang-mode
    '(6 nil warning "wrong number of arguments in format call" :checker erlang)))
 
+(flycheck-ert-def-checker-test erlang-rebar3 erlang error
+  (flycheck-ert-should-syntax-check
+   "language/erlang/rebar3/src/erlang-error.erl" 'erlang-mode
+   '(6 nil warning "wrong number of arguments in format call" :checker erlang-rebar3)
+   '(7 nil error "head mismatch" :checker erlang-rebar3)))
+
+(flycheck-ert-def-checker-test erlang-rebar3 erlang warning
+  (flycheck-ert-should-syntax-check
+   "language/erlang/rebar3/src/erlang-warning.erl" 'erlang-mode
+   '(6 nil warning "wrong number of arguments in format call" :checker erlang-rebar3)
+   '(7 nil error "head mismatch" :checker erlang-rebar3)))
```

Comment:
@joedevivo I'm aware of that but you should still see only errors for the current buffer?  If that was really the expected outcome of the test _all_ errors from _all_ files in the whole project would be highlighted in _all_ erlang buffers for the project, and that's clearly not desirable :)